tests/test_telemetry.py

```diff
@@ -0,0 +1,31 @@
+import os
+import unittest
+from unittest.mock import patch
```

Comment:
We dont use unittest and instead use pytest for testing. 

Please add tests using pytest. Also, can you please lint the code using `make format`?